app/controllers/weather_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WeatherController < ApplicationController
+
+  def forecast
```

Comment:
How could this action be used to perform a DOS attack on the system? What are some techniques to protect against this?